static_src/components/user_list.jsx

```diff
@@ -119,16 +121,25 @@ export default class UserList extends React.Component {
   }
 
   render() {
+    let savingContent;
+    let saving = <Saving />;
```

Comment:
I don't think you need both `saving` and `savingContent`. You could do:

```js
let saving;

if (this.state.saving) {
  saving = <Saving text={ this.state.saving } />;
}
```

If `saving` is undefined when you try and render it, React will ignore it.